Nanoradar NRA 24 UART connection information update (#6231)
ArduPilot/ardupilot_wiki: common/source/docs/common-rangefinder-nra24.rst

.. note::
    Support for this sensor is available in firmware versions 4.5 and later.

> The `NRA24 <http://en.nanoradar.cn/Article/detail/id/372.html>`__ is K-band radar commonly used for detecting height above terrain in all-weather conditions. Currently only connection via CAN is supported for one or more NRA24 modules. Serial (UART) versions of this Radar is only tested with single NRA24 module.

Reviewer: ```suggestion
The `NRA24 <http://en.nanoradar.cn/Article/detail/id/372.html>`__ is K-band radar commonly used for detecting height above terrain in all-weather conditions. Currently, only connection via CAN is supported for more than one NRA24 module. Serial (UART) versions of this Radar have only been tested with a single NRA24 module.
```

Setup the sensor through the NSM Software. In particular set the "ID" to a unique number if planning to use multiple NRA24 on a single CAN bus. You can also set the bitrate from the software.

.. note::

>     ArduPilot does not currently support same CAN rangefinders connected to different CAN ports. If you would like to use multiple CAN rangefinders together, connect them to the same bus.

Reviewer: ```suggestion
    ArduPilot does not currently support the same type CAN rangefinders connected to different CAN ports. If you would like to use multiple CAN rangefinders together, connect them to the same bus.
```

Set the following parameters

-  :ref:`CAN_P2_DRIVER <CAN_P2_DRIVER>` = 2 (to enable the second CAN driver)
-  :ref:`CAN_P2_BITRATE <CAN_P2_BITRATE>` = 500000 (Or as set in NSM tools)
-  :ref:`CAN_D2_PROTOCOL <CAN_D2_PROTOCOL>` = 14 (NanoRadar)
-  :ref:`RNGFND1_TYPE <RNGFND1_TYPE>` = 39 (NRA-24). Reboot after setting this.
-  :ref:`RNGFND1_MAX_CM <RNGFND1_MAX_CM>` = 19000 (i.e. 200m max range with 10m buffer)
-  :ref:`RNGFND1_MAX_CM <RNGFND1_MIN_CM>` = 50 (0.5m min range)
-  :ref:`RNGFND1_RECV_ID <RNGFND1_RECV_ID>` = ID of the sensor (0 to accept all CAN ids for distance)

Connecting via UART
===========================
For a serial connection TELEM1 and TELEM2 ports are tested. 

.. image:: ../../../images/nra24_uart_connections.png
    :target: ../_images/nra24_uart_connections.png

> Set the following parameters (For setup on TELEM1. Edit SERIAL2 for for TELEM2)

Reviewer: ```suggestion
Set the following parameters (Example for setup on TELEM1/SERIAL1)
```
